commcare_translations.py

```diff
@@ -54,5 +54,7 @@ def dumps(dct):
             val = u'\u00A0'
         # get rid of newlines
         val = val.replace('\n', '\\n')
+        # escape starting # character
+        val = re.sub('^#', '\#', val)
```

Comment:
Hey @ctsims I thought this would have come up before. Do you know if either of the following

```
myKey=# of patients
```

or

```
myKey=\# of patients
```

result in what you'd want (i.e. the value of "# of patients" for `myKey`)? Also is there anything important about the position of the `#`? this change is only escaping a `#` that starts a value so it wouldn't effect, say,

```
myKey2=total # of patients
```